src/components/errors.jsx

```diff
@@ -28,7 +28,10 @@ export default class ErrorMessage extends React.PureComponent {
     let msg = '';
     let back = false;
     let simhash = false;
-    const year = this.props.timestamp.substring(0, 4);
+    const year = ''
```

Comment:
Missing `;`

If you use `eslint`, you'll find out about things like that.

Reply:
Thank you. I have fixed this in the next commit.